docs/utilities/worksheet_layout_utility.rst

```diff
@@ -0,0 +1,62 @@
+Utility for extending Worksheet layouts
+-----------------------------
+
+Use this tool if you want to create and use your own custom worksheet layout views.
+
+
+1. Create class which implements ``IWorksheetLayouts`` marker interface and returns the list of your homemade layouts (as tuple of tuples) 
+
+.. code-block:: python
+
+    from zope.interface import implements
+    from bika.lims.interfaces import IWorksheetLayouts
+
+    class MyClassForWorksheetLayouts(object):
+        """ My worksheet layouts
+        """
+        implements(IWorksheetLayouts)
+
+        def getLayoutsList(self):
+            return (
+                ('my_first_ws_layout', 'My first WS layout'),
+            )
```

Comment:
Naming:
Rather name it `getLayouts` than `getLayoutsList`, because you are returning a tuple here.

Reason: Below you are using `+=` to append those values to another tuple. Returning here a list would raise this error:
```
TypeError: can only concatenate tuple (not "list") to tuple
```

Reply:
renamed to `getLayouts`